sklearn/linear_model/sag.py

```diff
@@ -89,10 +88,12 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1.,
     sample_weight : array-like, shape (n_samples,), optional
         Weights applied to individual samples (1. for unweighted).
 
-    loss : 'log' | 'squared'
-        Loss function that will be optimized.
-        'log' is used for classification, like in LogisticRegression.
-        'squared' is used for regression, like in Ridge.
+    loss : 'log' | 'squared' | 'multinomial'
```

Comment:
I at least use `row_norms` with `squared=True` frequently. Speaking of which let us take a step back.

    from sklearn.utils.extmath import row_norms
    def also_get_max_squared_sum(X):
        return row_norms(X, squared=True).max()
    X = np.random.rand(1000, 10000)
    %timeit get_max_squared_sum(X)
    100 loops, best of 3: 10.8 ms per loop
    %timeit also_get_max_squared_sum(X)
    100 loops, best of 3: 5.36 ms per loop
    X = np.random.rand(10000, 1000)
    %timeit get_max_squared_sum(X)
    100 loops, best of 3: 10.7 ms per loop
    %timeit also_get_max_squared_sum(X)
    100 loops, best of 3: 5.33 ms per loop

This should not matter much since it seems like it is done only once, but I feel it is unnecessary code duplication with almost no gain. (Unless I am overseeing something as usual)